3.analyze_data/compare_raw_mitocheckjump.ipynb

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.        if jump_random_sample_n &gt; 0:
</pre><p>Consider throwing an error if this condition isn't satisfied</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/JUMP-single-cell/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thank you for this suggestion! It may not be best practice, but given this function is defined locally to the notebook and is not expected to be used outside, it is self-contained and not meant for additional usage.

I believe that improving the function in this scenario in this specific way (to account for potential user error) is not a high value exercise. I probably, however, should do this more often when actively developing!